bin/sdl.cc

```diff
@@ -303,10 +303,25 @@ void run_sdl()
         queues.push_back(SDL::QueueAcc(devAcc));
     }
 
+    // Create objects that used to be global
+    uint16_t nModules;
+    uint16_t nLowerModules;
+    std::shared_ptr<SDL::modulesBuffer<SDL::Dev>> modulesBuffers = std::make_shared<SDL::modulesBuffer<SDL::Dev>>(devAcc);
```

Comment:
minor

```suggestion
    auto modulesBuffers = std::make_shared<SDL::modulesBuffer<SDL::Dev>>(devAcc);
```
could be less verbose